scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -127,5 +139,33 @@ def main():
   if return_error:
     sys.exit(1)
 
+  output_filename_1 = f"{build_dir}/tpch_output_1.json"
+  output_filename_2 = f"{build_dir}/tpch_output_2.json"
+
+  arguments = {}
+  arguments["--scale"] = ".01"
+  arguments["--chunk_size"] = "10000"
+  arguments["--queries"] = "'2,6,15'"
+  arguments["--runs"] = "10"
+  arguments["--output"] = output_filename_1
+  arguments["--dont_cache_binary_tables"] = "true"
+
+  benchmark = run_benchmark(build_dir, arguments, "hyriseBenchmarkTPCH", True)
+  benchmark.expect_exact(f"Writing benchmark results to '{output_filename_1}'")
+
+  close_benchmark(benchmark)
+  check_exit_status(benchmark)
+
+  arguments["--output"] = output_filename_2
+  arguments["--scheduler"] = True
+  benchmark = run_benchmark(build_dir, arguments, "hyriseBenchmarkTPCH", True)
+  benchmark.expect_exact(f"Writing benchmark results to '{output_filename_2}'")
+
+  close_benchmark(benchmark)
+  check_exit_status(benchmark)
```

Comment:
You added two runs of the benchmark. Please update the doc at the beginning of the file. Currently, it reads
> This test runs the binary hyriseBenchmarkTPCH with two different sets of arguments. [...]

Reply:
```
# This test runs the binary hyriseBenchmarkTPCH five times with various sets of arguments.
# During the first run, the shell output is validated using pexpect. After the first run, this test checks if an output
# file was created and if it matches the arguments.
# During the second run, the shell output is validated using pexpect and the test checks if all queries were
# successfully verified with sqlite.
# The third run checks the visualization of queries.
# The forth and fifth run are used to test the output of the compare_benchmarks.py script.
```

Ok?

Reply:
Nope. Changed it. I removed this comment altogether and added a short comment for each benchmark run.
I don't think we'll keep the comment above `main()` really up to date (what I just proved :D).

Reply:
Good choice.